src/mbed_os_tools/detect/platform_database.py

```diff
@@ -261,6 +261,9 @@
         u'190C': u'CY8CPROTO_064B0S3',
         u'190E': u'CY8CPROTO_062S3_4343W',
         u'190F': u'CY8CPROTO_064B0S1_BLE',
+        u'1910': u'CY8CKIT_064B0S2_4343W',
+        u'1911': u'CY8CKIT_064S0S2_4343W',
```

Comment:
The identifiers need to be 20 characters or less.

Reply:
@morser499 - As Graham highlights, the target name needs to be limited to 20 characters if possible.  This is a requirement comes from of our database which is essential to the online compiler and potentially other tools going forward.  I know that some other platforms have gotten in that violate this requirement, which we have not communicated very clearly.  We apologize for the inconvenience.  We just discussed this with our product manager.  It may be possible for us to increase this limit in the future.  But at this time, if you can shrink the target name by one character (go from 21 to 20 chars) that would be the quickest solution.

Reply:
Thanks @maclobdell for expanding on my brief comment. We intend to ensure that we remove this limitation in future versions of Mbed OS but at present there is an unknown consequence of going beyond this limit. Unfortunately, some names have breached this limit without us noticing, which has led to further confusion.

Reply:
I have reduced the length to 20 characters for the new boards.

Unfortunately this makes the names inconsistent with other boards which can lead to confusion.  Hopefully this limit can be addressed sometime soon.

Reply:
@madchutney any other issues with this or can we get it merged?

Reply:
I'll merge it in but please also ask PE to update the online entries for these product codes which are still listed as CYPRESS_PLACEHOLDER. Thanks.